src/bluetooth-fw/nimble/init.c

```diff
@@ -133,21 +137,14 @@ bool bt_driver_start(BTDriverConfig *config) {
   return true;
 }
 
-static void prv_ble_hs_stop_cb(int status, void *arg) {
-  xSemaphoreGive(s_host_stopped);
-}
-
 void bt_driver_stop(void) {
-  BaseType_t rc;
-  static struct ble_hs_stop_listener listener;
+  BaseType_t f_rc;
 
-  ble_hs_stop(&listener, prv_ble_hs_stop_cb, NULL);
-  rc = xSemaphoreTake(s_host_stopped, milliseconds_to_ticks(s_bt_stack_start_stop_timeout_ms)) ==
-  PBL_ASSERT(rc == pdTRUE, "NimBLE host stop timed out");
+  ble_hs_stop(&s_listener, prv_ble_hs_stop_cb, NULL);
```

Comment:
Can the first arg just be NULL, given we don't actually use it? https://github.com/apache/mynewt-nimble/blob/5ab995f8f262a9827f4ead69fb8934e8244c6825/nimble/host/src/ble_hs_stop.c#L191